conn/physic/units_test.go

```diff
@@ -35,8 +35,8 @@ func TestAngle_String(t *testing.T) {
 		{-1000 * Degree, "-1000°"},
 		{100000000000 * Degree, "100000000000°"},
 		{-100000000000 * Degree, "-100000000000°"},
-		{(9223372036854775807) * NanoRadian, "528460276055°"},
-		{(-9223372036854775807) * NanoRadian, "-528460276055°"},
+		{maxAngle * NanoRadian, "528460276055°"},
```

Comment:
maxAngle is already in NanoRadian, so this is technically an error to multiply by NanoRadian.